nixos/tests/xrdp.nix

```diff
@@ -18,6 +19,7 @@ import ./make-test-python.nix ({ pkgs, ...} : {
       environment.systemPackages = [ pkgs.freerdp ];
       services.xrdp.enable = true;
       services.xrdp.defaultWindowManager = "${pkgs.icewm}/bin/icewm";
+      hardware.pulseaudio.enable = true;
```

Comment:
Is pulseaudio required or does pipewire-pulse compat work

Reply:
I have to test tbh

It's because it requires a couple of pulseaudio plugins to work, plugin compatibility is another situation and upsteam doesn't mention at all about it.

Reply:
Did you get around to testing this?

Reply:
Almost

```
client # [   23.321148] wireplumber[1076]: RTKit error: org.freedesktop.DBus.Error.ServiceUnknown
client # [   23.323281] wireplumber[1076]: RTKit does not give us RTTimeUSecMax, using -1
client # [   23.344258] xrdp-sesman[1087]: Failure: No such entity
client # [   23.352406] xrdp-sesman[1067]: ? Can't load pulseaudio xrdp-sink
client # [   23.369724] xrdp-chansrv[1066]: [INFO ] sound_process_output_format:
client # [   23.373980] xrdp-chansrv[1066]: [INFO ]       wFormatNo       0
client # [   23.375868] xrdp-chansrv[1066]: [INFO ]       wFormatTag      WAVE_FORMAT_PCM
client # [   23.377956] xrdp-chansrv[1066]: [INFO ]       nChannels       2
client # [   23.383316] xrdp-chansrv[1066]: [INFO ]       nSamplesPerSec  44100
client # [   23.385645] xrdp-chansrv[1066]: [INFO ]       nAvgBytesPerSec 176400
client # [   23.389068] xrdp-chansrv[1066]: [INFO ]       nBlockAlign     4
client # [   23.391592] xrdp-chansrv[1066]: [INFO ]       wBitsPerSample  16
client # [   23.396210] xrdp-chansrv[1066]: [INFO ]       cbSize          0
client # [   23.399396] xrdp-chansrv[1066]: [INFO ] sound_process_output_format:
client # [   23.401068] xrdp-chansrv[1066]: [INFO ]       wFormatNo       1
client # [   23.405834] xrdp-chansrv[1066]: [INFO ]       wFormatTag      WAVE_FORMAT_PCM
client # [   23.410231] xrdp-chansrv[1066]: [INFO ]       nChannels       2
client # [   23.411796] xrdp-sesman[1088]: Failure: No such entity
client # [   23.412881] xrdp-chansrv[1066]: [INFO ]       nSamplesPerSec  22050
client # [   23.414157] xrdp-chansrv[1066]: [INFO ]       nAvgBytesPerSec 88200
client # [   23.415397] xrdp-chansrv[1066]: [INFO ]       nBlockAlign     4
client # [   23.416459] xrdp-chansrv[1066]: [INFO ]       wBitsPerSample  16
client # [   23.417953] xrdp-chansrv[1066]: [INFO ]       cbSize          0
client # [   23.419511] xrdp-chansrv[1066]: [INFO ] sound_process_training: round trip time 3
client # [   23.421796] xrdp-sesman[1067]: ? Can't load pulseaudio xrdp-source
client # [   23.487803] xrdp-sesman[1058]: rdpRRGetInfo:
```

Reply:
And it seems there is no way. Should I comment the test on passthru.tests or remove completely?

https://github.com/NixOS/nixpkgs/issues/252260

Reply:
Removed the tests, pipewire can be done in a follow up PR